post-til-private/src/test/java/no/difi/meldingsutveksling/ptp/MeldingsformidlerClientMain.java

```diff
@@ -59,7 +61,9 @@ public String getSpraakKode() {
 
             @Override
             public String getConversationId() {
-                return String.valueOf(UUID.randomUUID());
+                final String uuid = String.valueOf(UUID.randomUUID());
+                System.out.println("Melding sent med conversation id " + uuid);
```

Comment:
Tenker jeg lar den være siden jeg trenger den til å teste. Alternativt er å få inn loggerverktøy men det blir unødvendig mye arbeid for noe som ikke er produksjonskode.